### DISCUSSION TITLE
Change Indicator color of CtkRadiobutton

### DISCUSSION THREAD

**Discussion by AnwarHIS18** [TomSchimansky/CustomTkinter#1625] [Q&A] [answered]
can I customize the indicator color? currently, when the radiobutton is clicked, the indicator dot is white. can i change the color? thank you

**Comment by avalon60** [comment, marked as answer]
I'm not sure what you mean by the dot. The selected color of CTkRadioButton can be changed by modifying the fg_color property:

`CTkRadioButton.configure(fg_color=<color>)`

E.g.

`radio_button.configure(fg_color="#263b2b")`

The unselected color can be changed with:

`radio_button.configure(border_color="#3b2636")`
